### ISSUE TITLE
Sass support?

### ISSUE THREAD

**Issue by lyphtec** [andrewdavey/knapsack#17]
Hi,

Any chance of supporting Sass files as well?

Perhaps the "SassAndCoffee" nuget package (http://nuget.org/List/Packages/SassAndCoffee) can be used for this?

**Comment by andrewdavey** [comment]
Yes, it's on the to do list. We may be able to use SassAndCoffee directly, or cherry pick the Sass stuff out of there instead.
